### PR TITLE
chore(macros/WebExtAllExamples): update zh-* translation

### PR DESCRIPTION
**Pull request by skyclouds2001**
## Summary

part of the https://github.com/mdn/translated-content/issues/13881

/cc @yin1999 @JasonLamv-t @jasonren0403 

### Solution

---

### Before

image

### After

image

---

## How did you test this change?

observe the develop result

### CODE REVIEW

File: kumascript/macros/WebExtAllExamples.ejs [hunk 1]
```diff
@@ -17,15 +17,21 @@ const allExamples = await MDN.fetchWebExtExamples();
 const lang = env.locale;
 
 const s_name = mdn.localString({
-    "en-US": "Name"
+    "en-US": "Name",
+    "zh-CN": "名称",
```
Comment (skyclouds2001) [inline kumascript/macros/WebExtAllExamples.ejs:21]:
```suggestion
    "zh-CN": "名称 ",
```

File: kumascript/macros/WebExtAllExamples.ejs [hunk 2]
```diff
@@ -17,15 +17,21 @@ const allExamples = await MDN.fetchWebExtExamples();
 const lang = env.locale;
 
 const s_name = mdn.localString({
-    "en-US": "Name"
+    "en-US": "Name",
+    "zh-CN": "名称 ",
```
Comment (skyclouds2001) [inline kumascript/macros/WebExtAllExamples.ejs:21]:
```suggestion
    "zh-CN": "名称",
```

**Review by JasonLamv-t — APPROVED** [review]
LGTM.

**Review by yin1999 — APPROVED** [review]
LGTM. Thank you 🙏

**Review by jasonren0403 — APPROVED** [review]
👍

**Comment by jasonren0403** [comment]
/ping @mdn/core-dev for another look 👀 

addtionally, the 'Compare bundle size' check seems unrelevant for this commit

**Comment by PassionPenguin** [comment]
> /ping @mdn/core-dev for another look 👀
> 
> addtionally, the 'Compare bundle size' check seems unrelevant for this commit

same here https://github.com/mdn/yari/pull/11606#issuecomment-2381456099 🤔

**Review by caugner — APPROVED** [review]

**Pull request merged**